lib/Conversion/TorchToTcp/DataMovement.cpp

```diff
@@ -26,6 +28,75 @@ using namespace mlir::torch;
 using namespace mlir::torch::Torch;
 
 namespace {
+template <typename OpTy, typename OpAdaptor>
+LogicalResult prepareArgumentsForSlicingOp(OpTy op, OpAdaptor adaptor,
```

Comment:
If this is a copy-paste from [here](https://sourcegraph.com/github.com/llvm/torch-mlir@d65925a8b465d4a84be947d37197178d5c5cc6d2/-/blob/lib/Conversion/TorchToLinalg/DataMovement.cpp?L42), why not call it directly instead? Was it just not being exposed (via a header)? If so, maybe say "this was adapted from ..." add a hyperlink for future reference, and specify changes you made (if any).

Reply:
+1 to this. 

We could also consider updating TorchMLIR to expose this API via some header and use it directly here, instead of copying it. Doing this in a later PR is fine. If you plan to do that, please add a TODO to stating that.

Reply:
Good point. Added.